runtime/doc/lua.txt

```diff
@@ -2473,4 +2473,76 @@ trust({opts})                                             *vim.secure.trust()*
         • true and full path of target file if operation was successful
         • false and error message on failure
 
+
+==============================================================================
+Lua module: version                                              *lua-version*
+
+cmp({v1}, {v2}, {opts})                                        *version.cmp()*
+    Compares two strings ( `v1` and `v2` ) in semver format.
+
+    Parameters: ~
+      • {v1}    (string) Version.
+      • {v2}    (string) Version to be compared with v1.
+      • {opts}  (table|nil) Optional keyword arguments:
+                • strict (boolean): see `semver.parse` for details. Defaults
+                  to false.
+
+    Return: ~
+        (integer) `-1` if `v1 < v2`, `0` if `v1 == v2`, `1` if `v1 > v2`.
+
+eq({version_1}, {version_2})                                    *version.eq()*
+    Compares `version_1` and `version_2` and returns `true` if the are equal. If `version_1` or `version_2` are not able to be parsed, an error will be thrown.
```

Comment:
Too verbose. Similar for the gt/lt docstrings.

```suggestion
    Returns `true` if `version_1` are `version_2` are equal version strings.
```

Reply:
Equal versions, but not necessary equal strings? (thinking of `v1.2` and `v1.2.0` with `strict=false`)

Reply:
fixed